data-pipeline/csv-validation/petl-validation.py

```diff
@@ -0,0 +1,51 @@
+import petl as etl
```

Comment:
As an overall comment of the structure of this, I think a better structure would be something like:

```
data-pipeline/csv-validation/
-- validators/
-- exceptions/
```